### PR TITLE
Add script for building locally

### PR DESCRIPTION
**Pull request by celebdor**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+if [[ "$1" == "" ]]; then
+    BUILD_DIR="${HOME}/.local/share/coredns-machine-kubernetes-build"
+    if [[ ! -d "${PWD}/build" ]]; then
+        mkdir -p "$BUILD_DIR"
+    fi
+    mkdir -p "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes"
+    cp -r . "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes"
+    chcon -Rv -t container_file_t "$BUILD_DIR"
+    sudo podman run --rm \
```
Comment (bcrochet) [inline build.sh]: This still fails for me with a DNS error. Not really sure why.

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        /go/src/github.com/metalkube/coredns-machine-kubernetes/build.sh $(id -u)
+    if [[ -f "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes/coredns" ]]; then
+        cp "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes/coredns" .
+    fi
+else
+    echo "Building CoreDNS with machinekubernetes..."
+
+    go get -v github.com/coredns/coredns
+
+    pushd /go/src/github.com/coredns/coredns
```
Comment (bcrochet) [inline build.sh]: Shouldn't this (and any reference to /go) be $GOPATH? For building outside of a container?

Comment (celebdor) [reply build.sh]: in that base container image GOPATH is /go

File: build.sh (hunk 1 above)
Comment (celebdor) [reply build.sh]: Which step fails? sudo podman run?

**Pull request merged**